pickLayer/core/set_active_layer_tool.py

```diff
@@ -0,0 +1,137 @@
+# -*- coding: utf-8 -*-
+
+#  Copyright (C) 2022 National Land Survey of Finland
+#  (https://www.maanmittauslaitos.fi/en).
+#
+#
+#  This file is part of PickLayer.
+#
+#  PickLayer  is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  SpatialDataPackageExport is distributed in the hope that it will be useful,
```

Comment:
mistähän tämmönen nimi on jäännyt?

Reply:
My bad, my bad! Could you @rnousia please replace it from everywhere?

Reply:
Sure, thanks for spotting that!